style.css

```diff
@@ -1 +1,296 @@
-/*  Add your styling here */
+body {
+  box-sizing: border-box;
+}
+.navigation {
+  display: none;
+}
+.homepage-main {
+    column-gap: 10px;
+    row-gap: 10px;
+    display: grid;
+    /* grid-template-rows: repeat (5, auto); */
+    /*  grid-template-areas must follow the exact number of columns and rows defined above (4 columns and 4 rows)  */
+    grid-template-areas:
+      "hero"  
+      "welcome"         /* one for each column */
+      "feature"
+      "product"
+      "footer";           /* the dot is used to skip a column */
+  }
+  
+  .homepage-hero {
+    grid-area: hero;
+    display: grid;
+    background-size: cover;
+    background-image: url(/HTML-CSS-Coursework-Week3/assets/hero.jpeg);
+  }
+  .one-one {
+  justify-self: end;
+  padding: 15px;
+  }
+
+  .one-one div {
+  width: 35px;
+  height: 5px;
+  background-color: black;
+  margin: 6px 0;
+  }
+
+  .one-two {
+  justify-self: end;
+  align-self: flex-end;
+  font-style: italic;
+  font-size: 1.5rem;
+  padding-right: 10px;
+  padding-bottom: 10px;
+  }
+
+  .homepage-welcome {
+    grid-area: welcome;
+    padding: 15px;
+  }
+  
+  .homepage-feature {
+    grid-area: feature;
+    background-size: cover;
+  }
+
+  .homepage-feature img {
+    width: 100%;
+    object-fit: cover;
+  }
+  
+  .homepage-products {
+    grid-area: product;
+    gap: 0.5rem;
+    display: grid;
+    grid-template-areas:
+      "product-item1     product-item2"           /* one for each column */
+      "product-item3     product-item4"
+; 
+  }
+  .homepage-products img {
+    height: 8rem;
+    object-fit: cover;
+  }
+  .grid-item1 {
+    grid-area: product-item1;
+    justify-self: center;
+  }
+  .grid-item2 {
+    grid-area: product-item2;
+    justify-self: center;
+  }
+
+  .grid-item3 {
+    grid-area: product-item3;
+    justify-self: center;
+  }
+
+  .grid-item4 {
+    grid-area: product-item4;
+    justify-self: center;
+  }
+  
+  .homepage-footer {
+    background-color:rgb(194, 229, 241);
+    grid-area: footer;
+    padding-left: 15px;
+  }
+  .oleh {
+    text-align: center;
+  }
+  /* 
+    Now let's change it for larger screens 
+    (using quite small breakpoints here to allow for Codepen's small viewport)
+  */
+  
+  /* TABLET: */
+  @media screen and (min-width: 550px) {
+  
+    /*  
+      We only define rules which are DIFFERENT from the mobile version defined above. 
+    */
+    .one-one{
+      display: none;
+    }
+    .grid-item1 {
+      grid-area: product-item1;
+    }
+    .grid-item2 {
+      grid-area: product-item2;
+    }
+  
+    .grid-item3 {
+      grid-area: product-item3;
+    }
+
+    .grid-item4 {
+      display: none;
+    }
+    .homepage-products {
+      grid-area: product;
+      display: grid;
+      
+      grid-template-columns: repeat(3, 1fr);
+      grid-template-areas:
+        "product-item1  product-item2  product-item3" ;          /* one for each column */ 
+    } 
+
+     .contact {
```

Comment:
I think you missed an s here. the class is "contacts"

Reply:
I was using grid-template-areas only for small cakes which are part of main grid.
I used contact in separate grid for footer
<img width="349" alt="image" src="https://user-images.githubusercontent.com/12444326/201140012-2f7e2507-cc2f-4feb-9a43-a114c47bd910.png">